scripts/ci/check_compliance.py

```diff
@@ -27,19 +27,19 @@
 
 logger = None
 
-def git(*args, cwd=None):
+def git(*args, cwd=None, ignore_non_zero=False):
     # Helper for running a Git command. Returns the rstrip()ed stdout output.
     # Called like git("diff"). Exits with SystemError (raised by sys.exit()) on
-    # errors. 'cwd' is the working directory to use (default: current
-    # directory).
+    # errors if 'ignore_non_set' is set to False (default: False). 'cwd' is the
```

Comment:
```suggestion
    # errors if 'ignore_non_zero' is set to False (default: False). 'cwd' is the
```

Reply:
Thanks for spotting this mistake. Should be fixed now.